app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -48,7 +48,8 @@ def repackage_message
     end
 
     rescue_from RestClient::Exception do |e|
-      Rails.logger.error pp_exception(e)
+      option = [RestClient::Gone].include?(e.class) ? {with_backtrace: false} : {}
```

Comment:
I wonder whether we need a complete stack trace at all when we encounter an error from candlepin since the error happened outside the scope of our app. As long as we can tell which controller method and line (request) threw the error it's probably enough information.